.github/workflows/create-release-pr.yml

```
@@ -0,0 +1,145 @@
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Check for Rust file changes
+        uses: dorny/paths-filter@v3
+        id: changes
+        with:
+            filters: |
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)